src/sbtc-releases/sbtc-dev/get-started.md

```diff
@@ -0,0 +1,47 @@
+# Get Started with sBTC 0.1
+
+## Local setup
+### Requirements
+* install [clarinet](https://github.com/hirosystems/clarinet)
+* checkout main branch of repository [github.com/stacks-network/sbtc](https://github.com/stacks-network/sbtc)
+
+### Configuration
+1. Create configuration file `config.json` in a new folder `romeo/testing` with the following content:
+```
+{
+  "state_directory": "./state",
+  "contract": "../asset-contract/contracts/asset.clar",
+  "wif": "YOUR_WIF_CREDENTIALS",
+  "bitcoin_node_url": "http://devnet:devnet@localhost:8001/api",
+  "stacks_node_url": "http://localhost:3999",
+  "stacks_transaction_fee": 2000,
+  "bitcoin_transaction_fee": 2000,
+  "contract_name": "sbtc-alpha-romeo-testing"
+}
+```
+2. Create a folder `state` in `testing` folder
```

Comment:
Propose changing this and step 3 to describe what the system will do, rather than listing them as steps to complete.
```suggestion
This will create a folder named `state` within the `testing` folder, in addition to creating an empty `log.ndjson` file.
```

Reply:
This is actually not needed as it will automatically create this dir and everything it needs in it.